examples/avmnist_multimodal/configs/late_fusion.yaml

```diff
@@ -0,0 +1,4 @@
+# Created by Mohammod Suvon by modifying https://github.com/HaozhiQi/ISONet/blob/master/configs/IN1k-ISO18.yaml (under the MIT License)
```

Comment:
This is a bit overdoing. Not necessary. See other egs.

Reply:
fixed